docs/API.md

```diff
@@ -149,10 +149,10 @@ back at vehicle end.
 Use `skills` to describe a problem where not all tasks can be served
 by all vehicles. Job skills are mandatory, i.e. a job can only be
 served by a vehicle that has **all** its required skills. In other
-words: job `j` is eligible to vehicle `v` iff `j.skills` is included
```

Comment:
This is no a typo, but a feature. ;-) Intended to stand for "if and only if", https://en.wikipedia.org/wiki/If_and_only_if